scss/_settings_spacing.scss

```diff
@@ -117,7 +117,7 @@ $sp-after: (
 $spv-nudge: map-get($nudges, p) !default; // top: nudge; bottom: unit - nudge; result: height = exact multiple of base unit
 $spv-nudge-compensation: $sp-unit - $spv-nudge !default;
 $input-margin-bottom: $sp-unit * 4 - $spv-nudge * 2;
-$input-vertical-padding: calc($spv-nudge - 1px);
+$input-vertical-padding: calc($spv-nudge - $input-border-thickness);
```

Comment:
Even with this adjustment I'm not sure if the padding is correct. We have border only on the bottom, so it seems we should have calculated it separately for top and bottom padding?